tests/pages/add_item_page.py

```
@@ -0,0 +1,51 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support.wait import WebDriverWait
+from pages.base_page import BasePage
+from utilities.locators import CheckoutPageLocators, CommonLocators, InventoryPageLocators
```

Ruff (0.12.2)

`utilities.locators.CheckoutPageLocators` imported but unused

Remove unused import: `utilities.locators.CheckoutPageLocators`

(F401)